src/components/app/App.js

```diff
@@ -1,54 +1,56 @@
 import React from 'react';
 import { ShowContextProvider } from '../../contexts/ShowContext';
 import { Route, Switch } from 'react-router-dom';
-import Navbar from '../navbar/Navbar';
+// import Navbar from '../navbar/Navbar';
 import Container from '../container/Container';
-import Footer from '../footer/Footer';
+// import Footer from '../footer/Footer';
 
 function App() {
   return (
-    <ShowContextProvider>
+    <>
+     <ShowContextProvider>
       <Switch>
         <Route
           exact
-          path="gone-phishin/years"
-          render={() => {
+          path="/gone-phishin/years"
```

Comment:
On my pr. I changed the path to `"/"` We can change it back. I just am not sure how to direct it the above path without giving the years container a <Link>